packages/notte-sdk/src/notte_sdk/endpoints/agents.py

```diff
@@ -168,10 +170,46 @@ def start(self, **data: Unpack[AgentStartRequestDict]) -> AgentResponse:
         Returns:
             AgentResponse: The response obtained from the agent run request.
         """
+        format = "<level>{level: <8}</level> - <level>{message}</level>"
+        _ = logger.configure(handlers=[dict(sink=sys.stderr, level="INFO", format=format)])  # pyright: ignore[reportArgumentType]
+
         request = AgentStartRequest.model_validate(data)
         response = self.request(AgentsClient.agent_start_endpoint().with_request(request))
         return response
 
+    @staticmethod
+    def pretty_string(step: _AgentResponse, colors: bool = True) -> str:
```

Comment:
the logging is the same for the sdk and the agents in notte-agents ? Can we avoid the duplication to keep the logging consistent ?